addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14266,13 +14276,13 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Flatten TV show seasons" with label #20412
 #: system/settings/settings.xml
 msgctxt "#36144"
-msgid "Remove the TV show season node, toggles between 'If only one season' (default), 'Always' and 'Never'."
+msgid "Remove the TV show season node, toggles between \"If only one season\" (default), \"Always\" and \"Never\"."
```

Comment:
@da-anda Are you sure about that? I need to replace a lot of these and they will show up on skin labels.

I would like a definitive solution to replacing quotations in these areas and thers quite a few.
I will wait to chnage this until this is considered a bit and you and others reply @jjd-uk @phil65